src/Http/GraphRequest.php

```diff
@@ -293,8 +293,6 @@ public function getTimeout()
     *
     * @param mixed $client The client to use in the request
     *
-    * @throws GraphException if response is invalid; if 4xx/5xx is returned and $http_errors is true
```

Comment:
Actually this function has never thrown GraphException (before #437)

Reply:
Agreed, true.